rails6/en/chapter02-api.adoc

```diff
@@ -54,11 +54,11 @@ RESTful APIs must follow at least three simple guidelines:
 ** *PUT*: Updates a collection or member of the resources
 ** *DELETE*: Destroys a collection or member of the resources
 
-This might not be clear enough or may look like a lot of information to digest but as we move on with the tutorial, hopefully it’ll get a lot easier to understand.
+This might not be clear enough or may look like a lot of information to digest but as we move on with the tutorial, hopefully, it’ll get a lot easier to understand.
 
 === Routes, Constraints and Namespaces
 
-Before start typing any code, we prepare the code with git. We’ll be using a branch per chapter, upload it to GitHub and then merge it with master. So let’s get started open the terminal, `cd` to the `market_place_api` directory and type in the following:
+Before start typing any code, we prepare the code with git. We’ll be using a branch per chapter, upload it to GitHub and then merge it with the master branch. So let’s get started open the terminal, `cd` to the `market_place_api` directory and type in the following:
```

Comment:
What do you think about ?

```suggestion
Before start typing any code, we prepare the code with git. We’ll be using a branch per chapter, upload it to GitHub and then merge it on `master` branch. So let’s get started open the terminal, `cd` to the `market_place_api` directory and type in the following:
```

Reply:
It's usually said, "to merge one branch into another". 
But that's okay.